pkg/server/workspaces/remove.go

```diff
@@ -70,3 +70,35 @@ func (s *WorkspaceService) RemoveWorkspace(workspaceId string) error {
 	log.Infof("Workspace %s destroyed", workspace.Id)
 	return nil
 }
+
+func (s *WorkspaceService) ForceRemoveWorkspace(workspaceId string) error {
+	// This version of RemoveWorkspace ignores provider errors and continues with the deletion
+
+	workspace, err := s.workspaceStore.Find(workspaceId)
+	if err != nil {
+		return ErrWorkspaceNotFound
+	}
+
+	log.Infof("Destroying workspace %s", workspace.Id)
+
+	target, _ := s.targetStore.Find(workspace.Target)
+
+	for _, project := range workspace.Projects {
+		_ = s.provisioner.DestroyProject(project, target)
+	}
+
+	_ = s.provisioner.DestroyWorkspace(workspace, target)
```

Comment:
Please catch the error and log it with `log.Error`. It's important that we have feedback on the server side if the deletion process was preformed correctly.